pkg/apis/workflow/v1alpha1/common_test.go

```diff
@@ -0,0 +1,22 @@
+package v1alpha1
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCompareByPriority(t *testing.T) {
+
+	stepOne := WorkflowStep{Name: "A", Priority: 100}
```

Comment:
I'd have two sub-tests: one for sorting by priority and another by name

Reply:
Added.